Project.toml

```diff
@@ -1,6 +1,5 @@
 name = "JuliaReachTemplatePkg"
 uuid = "7e933142-fe23-11e8-0e3a-eb85481d2276"
-authors = ["mforets <mforets@gmail.com>"]
 version = "0.1.0"
```

Comment:
hmm or can you leave it empty `authors = []` ?

Reply:
We do not use that at the moment. If we change our mind, we should update it everywhere.